src/components/sidebar/helpers/generate-product-landing-nav-items.ts

```diff
@@ -1,5 +1,50 @@
 import { ProductData } from 'types/products'
 
+/**
+ * Generates the Sidebar menuItems for product landing pages
+ * (/waypoint, /vault, etc.).
+ *
+ * Written in a way that enables slow migration of products as they're released
+ * in beta via the `rootDocsPaths` property defined in
+ * `src/data/${productSlug}.json` files.
+ *
+ * @TODO remove the if-else once all products have a `rootDocsPaths` property
+ * defined in their `src/data/${productSlug}.json` files.
+ */
+export const generateProductLandingSidebarMenuItems = (
+  product: ProductData
+) => {
+  let menuItems
+
+  if (product.rootDocsPaths) {
+    const rootDocsNavItems = product.rootDocsPaths.map((rootDocsPath) => {
+      const { shortName, name, path } = rootDocsPath
+      return { title: shortName || name, fullPath: `${product.slug}/${path}` }
+    })
+    menuItems = [
+      ...rootDocsNavItems,
+      {
+        title: 'Tutorials',
+        fullPath: `/${product.slug}/tutorials`,
+      },
+      {
+        title: 'Install',
+        fullPath: `/${product.slug}/downloads`,
+      },
+      { divider: true },
+      ...product.sidebar.resourcesNavData,
+    ]
+  } else {
+    menuItems = [
+      ...product.sidebar.landingPageNavData,
```

Comment:
Should this `else` ever trigger? I imagine `rootDocsPaths` should be required now that we have it set up.

I am concerned that retaining `sidebar.landingPageNavData` might create ambiguity on *what* exactly is being used on the sidebar.  Maybe we remove that data & remove this else case and just set the `rootDocsPaths` data up as required?

Reply:
Context for anyone reading: Brandon and I chatted in-person and decided a `console.warn` would be good here as a breadcrumb during the process of migrating a product into beta.

@BrandonRomano how's this sound? The product name and slug will be dynamically filled in when the warning is logged to the console.

> Warning (generateProductLandingSidebarMenuItems): ${product.name} does not have a "rootDocsPaths" property. Please replace the "landingPageNavData" property with "rootDocsPaths" in "src/data/${product.slug}.json".